lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -42,16 +42,15 @@ class Map
 
 module Google
   module Auth
-    # Patch the gcloud command used by googleauth to avoid spamming stderr.
+    # Disable gcloud lookup in googleauth to avoid picking up its project id.
     module CredentialsLoader
-      # Set $VERBOSE to nil to mute the following warning:
-      # "warning: already initialized constant
-      # Google::Auth::CredentialsLoader::GCLOUD_CONFIG_COMMAND".
+      # Set $VERBOSE to nil to mute the "already initialized constant" warnings.
       warn_level = $VERBOSE
       begin
         $VERBOSE = nil
-        GCLOUD_CONFIG_COMMAND =
-          'config config-helper --format json --verbosity none'.freeze
+        GCLOUD_POSIX_COMMAND = '/bin/true'.freeze
```

Comment:
Can we add a comment and maybe a link to https://github.com/googleapis/google-auth-library-ruby/blob/f2f83b20bc1f32db0c90cf5cea0922d0558a35b0/lib/googleauth/credentials_loader.rb#L174 to clarify what we are doing?